### DISCUSSION TITLE
Clips does not play in HA

### DISCUSSION THREAD

**Discussion by Jurgonh** [blakeblackshear/frigate#14901] [Ask A Question] [answered]
In Home Assistant, clips are no longer playing when i click in the clip. The time is stuck on 0,0. It has been working before. Since i updated both Home Assistant, Frigate and Frigate card to new versions i dont know which component that causes this,  or if there is any settings that i must change. The problem is the samt both on iPad in the companion app and in broser on my computer. Hope anyone can help. 

https://github.com/user-attachments/assets/574c906a-95d2-43f7-a961-c90fdf41e46e

**Comment by NickM-27** [comment, marked as answer]
It's a bug introduced by home assistant

**Reply by Jackdawson111** [reply]
I am experiencing the same issue after performing the 2024.11.1 Home Assistant Core update.

**Reply by Zixim** [reply]
meanwhile things aren't any better with 2024.11.2 & 2024.11.3 :-1:

**Comment by Trumaco** [comment]
A mi tambien me pasa lo mismo!!
